README.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+
+<html lang="en-US">
+    <head>
+        <title>README file</title>
+
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link href="@css/documentation_style.css" rel="stylesheet" type="text/css">
+    </head>
+
+    <body>
+      <p><a name="readme-top"></a></p>
+      
+      <!--
+      !!! IMPORTANT !!!
+      This README is an example of how you could professionally present your codebase. 
+      Writing documentation is a crucial part of your work as a professional software developer and cannot be ignored. 
+      
+      You should modify this file to match your project and remove sections that don't apply.
+      
+      REQUIRED SECTIONS:
+      - Table of Contents
+      - About the Project
+        - Built With
+        - Live Demo
+      - Getting Started
+      - Authors
+      - Future Features
+      - Contributing
+      - Show your support
```

Comment:
- [ ] Commented code makes your codebase look clustered and may result to errors and confusion sometimes. Kindly delete this to keep your codebase clean 👍